inference-engine/src/vpu/myriad_plugin/myriad_executor.cpp

```diff
@@ -261,13 +261,11 @@ DevicePtr MyriadExecutor::openDevice(std::vector<DevicePtr>& devicePool,
             });
 
         // Return mock device. If try infer with it, exception will be thrown
-        if (availableDevices.empty() && config.platform() != NC_ANY_PLATFORM) {
+        if (availableDevices.empty()) {
             DeviceDesc device;
             device._platform = config.platform();
```

Comment:
What should be the value of `_platform` in that case? It matters because there is a check on `_platform` that it's not `NC_ANY_PLATFORM` which is the case as I understand if user did not specified specific revision (and that's definitely the case since there should not be such option)